.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -38,6 +38,7 @@
         - [ ] Verify that the `PrivateSourceBuiltArtifactsPackageVersion` in `eng/Versions.props` matches N-1 release artifacts.
         - [ ] Update the `global.json` SDK version to N-1: the latest released "3.1.XYY" SDK version, where "3.1.X" matches the SDK we're building.
         - [ ] Update the `eng/Versions.props` with the latest product versions.
+        - [ ] Verify that all repos that are not pinned were updated, or that we have confirmed that there are not expected to update for this release.
```

Comment:
I have the same question here:  If I look in Version.Details.xml for rel/5.0, I see one repo pinned (aspnet-xdt). What this is saying is that all other repos should be updated? How should we confirm that the ones that are not updated are not expected to change for this release?

Reply:
Also added the list of expected repos here.